.github/ISSUE_TEMPLATE/config.yaml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
```

Comment:
prefer to change it as `false`, always using templates to create issue

Reply:
changed.